plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -228,26 +233,32 @@ func BalancedResourceAllocation(pod *api.Pod, nodeNameToInfo map[string]*schedul
 func calculateBalancedResourceAllocation(pod *api.Pod, node api.Node, nodeInfo *schedulercache.NodeInfo) schedulerapi.HostPriority {
 	totalMilliCPU := nodeInfo.NonZeroRequest().MilliCPU
 	totalMemory := nodeInfo.NonZeroRequest().Memory
+	totalNvidiaGPU := nodeInfo.NonZeroRequest().NvidiaGPU
 	score := int(0)
 	// Add the resources requested by the current pod being scheduled.
 	// This also helps differentiate between differently sized, but empty, nodes.
 	for _, container := range pod.Spec.Containers {
-		cpu, memory := priorityutil.GetNonzeroRequests(&container.Resources.Requests)
+		cpu, memory, nvidiaGPU := priorityutil.GetNonzeroRequests(&container.Resources.Requests)
 		totalMilliCPU += cpu
 		totalMemory += memory
+		totalNvidiaGPU += nvidiaGPU
 	}
 
 	capacityMilliCPU := node.Status.Allocatable.Cpu().MilliValue()
 	capacityMemory := node.Status.Allocatable.Memory().Value()
+	// TODO: do we need this?
+	// capacityNvidiaGPU := node.Status.Allocatable.NvidiaGPU().Value()
 
 	cpuFraction := fractionOfCapacity(totalMilliCPU, capacityMilliCPU)
 	memoryFraction := fractionOfCapacity(totalMemory, capacityMemory)
+	// TODO: Do we need this?
+	// nvidiaGpuFraction := fractionOfCapacity(totalNvidiaGPU, capacityNvidiaGPU)
```

Comment:
I think the scheduling scores should not change at all as a result of adding this alpha code. Once we have multiple users of GPUs reporting their scheduling issues, then we might consider how to weight GPUs versus other resources.